### PR TITLE
Fix test result conclusion in DTR document

### LINKED ISSUE #68: Test conclusion in the DTR shows a major discrepancy although all tests are successful

### PR DESCRIPTION
**Pull request by metmajer**
Closes #68

### CODE REVIEW

File: src/org/ods/usecase/LeVADocumentUseCase.groovy [hunk 1]
```diff
@@ -83,6 +83,9 @@ class LeVADocumentUseCase extends DocGenUseCase {
     }
 
     private Map computeTestDiscrepancies(String name, List jiraTestIssues) {
+
+        // echo "nifl::name: ${name}; jiraTestIsues: ${jiraTestIssues.join(",")}"
```
Comment (metmajer) [inline src/org/ods/usecase/LeVADocumentUseCase.groovy]: @georgfedermann please remove stale comments

File: src/org/ods/usecase/LeVADocumentUseCase.groovy [hunk 2]
```diff
@@ -334,6 +337,9 @@ class LeVADocumentUseCase extends DocGenUseCase {
     }
 
     String createDTR(Map project, Map repo, Map data) {
+
+        // echo "nifl::createDTR being executed with: project: ${project.toMapString()}; repo: ${repo.toMapString()}; data: ${data.toMapString()}"
```
Comment (metmajer) [inline src/org/ods/usecase/LeVADocumentUseCase.groovy]: please remove stale comments

**Review by metmajer — COMMENTED** [review]
Nice one @georgfedermann! Some minor polishing requested.

File: src/org/ods/usecase/LeVADocumentUseCase.groovy (hunk 1 above)
Comment (georgfedermann) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: done

File: src/org/ods/usecase/LeVADocumentUseCase.groovy (hunk 2 above)
Comment (georgfedermann) [reply src/org/ods/usecase/LeVADocumentUseCase.groovy]: done

**Comment by metmajer** [comment]
The screenshots show that a successful test does not get interpreted as a major discrepancy (in other words a failure). @georgfedermann nice job!

Screenshot 2020-02-13 at 09 50 36
Screenshot 2020-02-13 at 09 50 41

**Pull request merged**